app/client/src/features/events/Moderation/ManageQuestions/CurrentQuestionCard.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/indent */
```

Comment:
Hmm, is there a reason we're moving current question card outside of manage questions?

Reply:
Ah, I see after looking at the figma design.  Is the question carousel getting removed?

Reply:
I think so based on the Figma design, so I made that current question card component.